sphinxcontrib/openapi/openapi30.py

```diff
@@ -119,6 +118,10 @@ def _parse_schema(schema, method):
         return [_parse_schema(schema['items'], method)]
 
     if schema_type == 'object':
+        if 'example' in schema:
+            example = schema.get('example')
+            return collections.OrderedDict(example)
```

Comment:
According to OpenAPI 3.0 spec, the `example` of schema:

> A free-form property to include an example of an instance for this schema. To represent examples that cannot be naturally represented in JSON or YAML, a string value can be used to contain the example with escaping where necessary.

Which means, we cannot cast it to OrderedDict.